packages/babel-plugin-async-load/src/index.js

```diff
@@ -0,0 +1,58 @@
+const lazyLoadTemplate = `
+var COMPONENT_NAME = wp.compose.lazyLoad( SCRIPTS, STYLES, BASE_URL)( WRAPPED_COMPONENT );
+`;
+
+export default function( babel ) {
+	const { types: t, template } = babel;
+
+	return {
+		visitor: {
+			Program: function( path, state ) {
+				// 1. Find occurences to replace
+				const mappings = [];
+				const importVisitor = {
+					ImportSpecifier: function( subPath ) {
+						state.opts.components.forEach( ( componentConfig ) => {
+							if (
+								subPath.parent.source.value === componentConfig.module &&
+								subPath.node.imported.name === componentConfig.component
+							) {
+								// Add the Wrapper
+								const buildRequire = template( lazyLoadTemplate );
```

Comment:
How heavy an operation is `template` call ? Should it be called once at the assignment of the `lazyLoadTemplate` variable itself? Or maybe a Lodash `_.once` ?